tests/conftest.py

```diff
@@ -29,7 +29,7 @@ def f():
 
 def pytest_runtest_setup(item):
     global mcache_server_option
-    mcache_server_option = item.config.getoption('--memcached')
+    mcache_server_option = item.config.getoption('--memcached', 'localhost')
```

Comment:
I tried to change the tests to connect to localhost here, but they still end up running docker. I had to hardcode it in the mcache_server() function below.

I don't know if you want to fix up this docker code yourself, or just remove all of it.